Makefile

```diff
@@ -4,7 +4,7 @@ SHELL=/bin/bash
 export CFLAGS += -std=c11 -Wall
 
 .PHONY: all
-all: VERSION.full bin/version.h bin/version.sh
+all: VERSION.full bin/version.h bin/version.sh charliecloud.spec
```

Comment:
Right now the package builds are not invoked by `make` at the topic level. If we change this, I'd like to make the change mindfully. What's your reasoning?